### PR TITLE
Merak-Compute Concurrency

### PR DESCRIPTION
**Pull request by phudtran**
This PR adds the following

- Concurrency for VM creation
- Concurrency for VM Delete
- Async reply to Scenario Manager
- Refactor and cleanup compute handler
- Fix an issue where agent did not return an error on bad HTTP response from Alcor
- Change to new temporal client from deprecated one
- Additional detailed agent logging
- Fix Makefile issue where compute-vm-worker wasn't compiled

### CODE REVIEW

**Comment by phudtran** [comment]
#82

File: services/merak-compute/handler/create.go [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+	log.Println("Operation Create")
+	return_vms := []*pb.InternalVMInfo{}
+	// Add pods to DB
+	for n, pod := range in.Config.Pods {
+		if err := RedisClient.HSet(
+			ctx,
+			pod.Id,
+			"name", pod.Name,
+			"ip", pod.ContainerIp,
```
Comment (cj-chung) [inline services/merak-compute/handler/create.go:48]: @phudtran Should we consider the redis db lock here if we have multiple compute instances running in the k8s cluster?

Comment (phudtran) [reply services/merak-compute/handler/create.go:48]: Not as of now since we only send one big protobuf message per run.

**Review by cj-chung — APPROVED** [review]
LGTM

**Pull request merged**